src/api/reactivity-core.md

```diff
@@ -52,15 +52,15 @@ Takes a getter function and returns a readonly reactive [ref](#ref) object for t
   ```ts
   // read-only
   function computed<T>(
-    getter: () => T,
+    getter: (oldValue: T | undefined) => T,
```

Comment:
Is it correct that this is `: T | undefined` instead of `?: T`, if I remember correctly there might be a TS compiler options so that `| undefined` means that it needs to explicitly pass a variable that can be undefined, instead of not passing a variable at all.  
However, because this is a callback, it is not affected by this compile option.

Reply:
This function is not passed by the user but provided by the framework, so the argument will always be present.